frontend/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,13 @@
+import {
+  Checkbox as ChakraCheckbox,
+  CheckboxProps,
+  forwardRef,
+} from '@chakra-ui/react'
+
+export const Checkbox = forwardRef<CheckboxProps, 'input'>((props, ref) => {
+  return (
+    <ChakraCheckbox {...props} ref={ref}>
+      {props.value}
```

Comment:
Shouldn't we pass in props.children instead?

Reply:
and if that's the case, we can use Checkbox directly, from Chakra without doing anything more right?

```ts
export const Checkbox = ChakraCheckbox
```

Reply:
I wanted the way we specify the value/child of `Checkbox` to be consistent for `Checkbox` and `CheckboxOthers`. For `CheckboxOthers`, it will be difficult to distinguish the text from the others component if the value is passed in through the child so I standardised it by always using the "value" prop. There's a tradeoff here though, because we can't decouple the submission value and the value that is displayed. Let me know if there's a better way of doing this :)

Reply:
Its not mutually exclusive right? The checkbox just needs a `value` prop, the child does not matter afaik.

e.g.

```ts
<Checkbox value="a">
  Can be anything here
</Checkbox>
```

Also doesn't matter for checkbox others right?

```ts
```